package.json

```diff
@@ -87,7 +85,6 @@
         "next": "^10.2.3",
         "next-i18next": "^7.0.1",
         "numeral": "^2.0.6",
-        "path": "^0.12.7",
```

Comment:
This one stuck out to me as well because it's used quite a bit over on the server side, but it looks like it's built into node if I'm reading this right: https://nodejs.org/api/path.html